pages/glossary/visual-context.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Visual context
+key: visual-context
+---
+
+The visual context of some piece of information (_e.g._ an element) is everything that is visually or logically located near it. Only [visible](#visible) information may be part of the visual context.
+
+Visual context may include, and is not limited to, headings, text in the same sentence or paragraph, …
```

Comment:
I do believe that paragraphs and headings have a programmatically determinable relationship. Is there an overlap between programmatically determined context and visual context? If so I would mention that in a note and I would add headings to the examples of both.

Reply:
Correct. I do not know why I missed headings and paragraph in that list 😞 
(and yes, there is an overlap between visual and prog. det. context and I see no problem with that)